src/analysis_results.jl

```diff
@@ -0,0 +1,18 @@
+using GraphPlot
+
+sets, graph, F, solution = run_model()
+
+# plot flow time series
+from_flow = "Valhalla_Fuel_cell"
+to_flow = "Valhalla_E_balance"
+p = show_flow_time_series(from_flow, to_flow, 1, sets.time_steps, solution.flow)
+display(p)
+
+# plot graph with edge values
+rp = 1
+time = 1
+edge_labels = Dict(f => solution.flow[f, rp, time] for f in F)
+
+edge_label_list = [edge_labels[(src(e), dst(e))] for e in edges(graph)]
+g = gplot(graph; edgelabel = edge_label_list)
+display(g)
```

Comment:
Should I ignore this file?

Reply:
Hi @abelsiqueira ,
The rationale for having this file is given above
> In addition, I realized there is no existing function for running the model and deriving the results (apart from the test). For this purpose, also a file in src has been added. This may related to https://github.com/TulipaEnergy/TulipaEnergyModel.jl/issues/209. Is creating such a file a desired way to do so?

The line 16-18 is just some early codes for plotting the graph with edge values, which are not working yet.

I'm not sure which you'd like to dump, but feel free to do so as I won't be able to follow it up soon. Thanks.